Update fields_to_export parameter documentation for dynamic rate limiting (#10205)
braze-inc/braze-docs: _docs/_api/endpoints/export/user_data/post_users_identifier.md

| `braze_id`         | Optional | String                                                        | Braze identifier for a particular user.                                                      |
| `email_address`    | Optional | String                                                        | Email address of user.                                                                       |
| `phone`            | Optional | String in [E.164](https://en.wikipedia.org/wiki/E.164) format | Phone number of user.                                                                        |

> | `fields_to_export` | Optional* | Array of strings                                              | Name of user data fields to export.<br><br>*This field is required to automatically use the faster rate limit. If omitted, the default rate limit of 250 requests per min will be used. |

Reviewer: ```suggestion
| `fields_to_export` | Optional* | Array of strings                                              | Name of user data fields to export.<br><br>*This field is required to automatically use the faster rate limit. If omitted, the default rate limit of 250 requests per min will be used instead. |
```

> | `fields_to_export` | Optional* | Array of strings                                              | Name of user data fields to export.<br><br>*This field is required to automatically use the faster rate limit. If omitted, the default rate limit of 250 requests per min will be used instead. |

Reviewer 2: 40 requests/sec

> | `fields_to_export` | Optional* | Array of strings                                              | Name of user data fields to export. If you want to automatically be upgraded to the faster rate limit, the use of this field is required. If you omit this field, your requests will be on the default 250 requests/min limit. |

Reviewer: ```suggestion
| `fields_to_export` | Optional* | Array of strings                                              | Name of user data fields to export.<br><br>*This field is required to use the faster rate limit. If omitted, the default rate limit of 250 requests/min will be applied. |
```
